lib/compute/jenkins-main-node.ts

```diff
@@ -330,8 +330,7 @@ export class JenkinsMainNode {
 
       // install all the list of plugins from the list and restart (done in same command as restart is to be done after completion of install-plugin)
       // eslint-disable-next-line max-len
-      InitCommand.shellCommand(`java -jar /jenkins-cli.jar -s http://localhost:8080 -auth @${JenkinsMainNode.JENKINS_DEFAULT_ID_PASS_PATH} install-plugin ${JenkinsPlugins.plugins.join(' ')} `
-      + ` && java -jar jenkins-cli.jar -s http://localhost:8080 -auth @${JenkinsMainNode.JENKINS_DEFAULT_ID_PASS_PATH} restart`),
+      ...JenkinsMainNode.createPluginInstallCommands(JenkinsPlugins.plugins),
```

Comment:
How to understand this leading up triple dot?
I only used it in arguments or arrays, not leading up to a call.

Reply:
It expands array into individual elements, which are composed into an array returned by the `configElements` function

Check out this stackoverflow comment that provide more context https://stackoverflow.com/a/61683996

Reply:
You can play around with it in your browser's console, these arrays are all the same. 
```javascript
[1, 2, 3, 4] == [1, ...[2, 3], 4] == [1, ...[2, ...[3]], 4]
```

Reply:
I got that, but why is it spreading on JenkinsMainNode?
I can understand if this is spread on plugins list.
But mainnode is only having one entry, right?

Reply:
This behavior is verified with `test/compute/jenkins-main-node.test.ts` updates